jetty-unixsocket/jetty-unixsocket-client/src/test/java/org/eclipse/jetty/unixsocket/UnixSocketTest.java

```diff
@@ -73,8 +74,9 @@ public void before() throws Exception
             sockFile = Files.createTempFile(Paths.get(unixSocketTmp), "unix", ".sock");
         else
             sockFile = Files.createTempFile("unix", ".sock");
+        if (sockFile.toAbsolutePath().toString().length() > UnixSocketConnector.MAX_UNIX_SOCKET_PATH_LENGTH)
+            sockFile = Files.createTempFile(Paths.get("/tmp"), "unix", ".sock");
```

Comment:
What happens if `/tmp` doesn't exist?

Reply:
Perhaps a lookup of tmp dir using the following ...

``` java
System.getenv("TMPDIR");
System.getenv("TMP");
Files.exists("/var/tmp");
Files.exists("/tmp");
```

Reply:
All that seems too complicated for a test case.
I have just added an assumption that `/tmp` exists; if it does not, the test is skipped.